pkgs/development/typst-packages/oxifmt.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  buildTypstPackage,
+  fetchFromGitHub,
+}:
+
+buildTypstPackage rec {
+  pname = "oxifmt";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "PgBiel";
+    repo = "typst-oxifmt";
+    rev = "v${version}";
+    hash = "sha256-OJpkUsoAOHHn7T2O+wICW4nLJ+epkAOfe5R1pMhv1MQ=";
+  };
+
+  meta = with lib; {
+    homepage = "https://github.com/PgBiel/typst-oxifmt";
+    description = "Convenient Rust-like string formatting in Typst (previously \"typst-strfmt\")";
```

```suggestion
    description = "Convenient Rust-like string formatting in Typst";
```
Information like a package's previous name is best left to `longDescription`, or omitted entirely.